Library/Homebrew/formula.rb

```diff
@@ -1338,6 +1338,11 @@ def std_cmake_args
     args
   end
 
+  # Standard parameters for Go builds.
+  def std_go_args
+    ["-trimpath", "-o", bin/name]
```

Comment:
Presumably if you pass another `-o` here it'll be used instead?

Reply:
Answered here: https://github.com/Homebrew/brew/pull/6923#discussion_r367653075